src/Transition/UtilityRoutines.f90

```diff
@@ -895,6 +895,73 @@ SUBROUTINE ShowErrorMessage(ErrorMessage,OutUnit1,OutUnit2)
 
 END SUBROUTINE ShowErrorMessage
 
+
+REAL FUNCTION GetSatVapPressFromDryBulb(Tdb) RESULT(retval)
+
+    ! Function to compute saturation vapor pressure in [kPa]
+    !  ASHRAE Fundamentals handbood (2005) p 6.2, equation 5 and 6
+    !    Valid from -100C to 200 C
+
+  REAL, INTENT(IN) :: Tdb ! Dry bulb temperature [degC]
+  REAL, PARAMETER :: C1 = -5674.5359
+  REAL, PARAMETER :: C2 = 6.3925247
+  REAL, PARAMETER :: C3 = -0.009677843
+  REAL, PARAMETER :: C4 = 0.00000062215701
+  REAL, PARAMETER :: C5 = 2.0747825E-09
+  REAL, PARAMETER :: C6 = -9.484024E-13
+  REAL, PARAMETER :: C7 = 4.1635019
+  REAL, PARAMETER :: C8 = -5800.2206
+  REAL, PARAMETER :: C9 = 1.3914993
+  REAL, PARAMETER :: C10 = -0.048640239
+  REAL, PARAMETER :: C11 = 0.000041764768
+  REAL, PARAMETER :: C12 = -0.000000014452093
+  REAL, PARAMETER :: C13 = 6.5459673
+
+  REAL TK
+  TK = Tdb + 273.15      ! Converts from degC to degK
+
+  IF (TK <= 273.15) THEN
+         retval = EXP(C1/TK + C2 + C3*TK + C4*TK**2 + C5*TK**3 + C6*TK**4 + C7*LOG(TK)) / 1000
```

Comment:
Just to **ensure** no shenanigans, I verified the nature of logarithms in each language:

 - Python math.log(x[, base]): [With one argument, return the natural logarithm of x (to base e).](https://docs.python.org/2/library/math.html#math.log)
 - Fortran LOG(x): [natural logarithm of x](http://www.cs.mtu.edu/~shene/COURSES/cs201/NOTES/chap02/funct.html)
 
OK, we're good, carry on.